### ISSUE TITLE
Installation of GENIE3 into R

### ISSUE THREAD

**Issue by prajayshah** [aertslab/GENIE3#3]
Hi I've tried installing GENIE3 into R, but I keep getting hit with this:

```
Warning message:
package ‘GENIE3’ is not available (for R version 3.4.0) 
```

Any ideas why this is a problem?

Thanks,

Prajay

**Comment by s-aibar** [comment]
If you are trying to install GENIE3 from Bioconductor with `biocLite("GENIE3")`, the problem is probably the R version. The current version of Bioconductor requires R >=3.4.2, but you have 3.4.0.

You can try to install it directly from the source with:
`install.packages("GENIE3_1.0.0.tar.gz", repos=NULL, type="source")`
or from this Github repository with:
`devtools::install.github(...)`

**Comment by alnf** [comment]
I have problem now with R 3.4.3:

`package ‘GENIE3’ is not available (for R version 3.4.3)`

While for 3.4.1 I was able to install GENIE3 using biocLite()

**Comment by RobDer00** [comment]
I am also having an issue with 3.4.3
if I go back to version 3.4.2, I get the same message:
package ‘GENIE3’ is not available (for R version 3.4.2)
